app/models/vestal_version.rb

```diff
@@ -0,0 +1,11 @@
+# This is a _temporary_ class to make up for the fact that the
+# vestal versions gem is no longer in the project. It's here to
+# allow access to the vestal data.
+# once that data is no longer needed, this class can be removed
+# along with the has_many clause in OrderDetail
+
+class VestalVersion < ActiveRecord::Base
+
+  belongs_to :versioned, polymorphic: true
```

Comment:
Can we add `serialize :modifications`? That's what converts the nasty stored text into something readable.